clickhouse/datadog_checks/clickhouse/config_models/instance.py

```diff
@@ -58,6 +58,7 @@ class Config:
     tls_verify: Optional[bool]
     use_global_custom_queries: Optional[str]
     username: Optional[str]
+    clustername: Optional[str]
```

Comment:
This file is autogenerated, so if you want to add `clustername` as a new configuration option, you can add it [here](https://github.com/DataDog/integrations-core/blob/master/clickhouse/assets/configuration/spec.yaml), and then run `ddev validate config clickhouse -s` and `ddev validate models clickhouse -s` to regenerate this file and the `conf.yaml.example` file.

Reply:
Ah, got it. let me try. Thank you!

Reply:
I got ```bash-3.2$ ddev validate config clickhouse -s 
Validating default configuration files for 1 checks...
Unsupported check: clickhouse or manifest_version: None```